app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFields.scala

```diff
@@ -21,7 +21,7 @@ import java.util.UUID
 import uk.gov.hmrc.apisubscriptionfields.model.{Fields, SubscriptionIdentifier}
 
 object SubscriptionFields {
-  def apply(id: SubscriptionIdentifier, fieldsId: UUID, customFields: Fields) : SubscriptionFields = SubscriptionFields(id.encode(), fieldsId, customFields)
+  def apply(id: SubscriptionIdentifier, fieldsId: UUID, customFields: Fields) : SubscriptionFields = new SubscriptionFields(id.encode(), id.applicationId.value, fieldsId, customFields)
```

Comment:
could you rename `customFields` to `fields`?

Reply:
done